src/scripts/build-docs.sh

```diff
@@ -5,6 +5,6 @@ if [ -z "$BASH_VERSION" ]; then
 fi
 SCDIR=$(dirname "$(readlink -f "${BASH_SOURCE[0]}")")
 SCDIR=$(realpath $SCDIR)
-pushd "$SCDIR" > /dev/null || exit
+pushd "$SCDIR/../.." > /dev/null || exit
```

Comment:
I don't see this in the port source? What is this doing?

Reply:
Still curious about this one.

Reply:
Fixed for improved readability.

Reply:
Reverting the change and moving to separate PR